src/Main.elm

```diff
@@ -60,7 +60,9 @@ type alias Config =
     { viewMode : ViewMode
     , formFields : Array FormField
     , formValues : Json.Encode.Value
-    , shortTextTypeList : List ( String, Dict String String )
+
+    -- List because order matters
+    , shortTextTypeList : List CustomElement
     }
```

Comment:
comment remind self why we want `shortTextTypeList` to be provided as a `List` not `Dict`: order matters since we are rendering them in the dropdown

![Screenshot 2024-09-14 at 2 52 19 PM](https://github.com/user-attachments/assets/5e978365-d311-43b9-a897-0d314ff3b59b)